source/Octopus.Tentacle.Contracts/Octopus.Tentacle.Contracts.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RootNamespace>Octopus.Tentacle.Contracts</RootNamespace>
+    <AssemblyName>Octopus.Tentacle.Contracts</AssemblyName>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputPath>bin\</OutputPath>
+    <TargetFrameworks>net452;netcoreapp3.1</TargetFrameworks>
```

Comment:
Should these be updated as they're EOL?

Reply:
Probably yes, but that is a separate discussion about what we are willing to support.